### PR TITLE
Added e2e tests for volunteer and student

### PR DESCRIPTION
**Pull request by StewartWallace1115**
Links
-----
- Issue: https://github.com/UPchieve/web/issues/273
- Server repo PR: _link to companion PR in server repo (or remove this line)_
- Dependency PRs: _links to PRs that must be merged prior this one (or remove this line)_

Description
-----------
- Removed student-login.js test and replaced it with login-tests. 
- Login-tests running e2e testing that both a volunteer and student can login and log out. It also tests that a user cannot login using invalid email or password. 
-ToDo: Add API checks. 

Developer self-review checklist
-------------------------------
- [ ] Potentially confusing code has been explained with comments
- [x] No warnings or errors have been introduced; all known error cases have been handled
- [ ] Any appropriate documentation (within the code, README.md, docs, etc) has been updated
- [x] All edge cases have been addressed

### CODE REVIEW

File: tests/e2e/specs/login-tests.js [hunk 1]
```diff
@@ -59,67 +54,63 @@ describe("Successful Log in and log out", () => {
...
       cy.location("pathname").should("eq", "/dashboard");
     });
-  
 
     it("Should log out successfully", () => {
-      cy.contains("Log out")
-        .click();
+      cy.contains("Log out").click();
 
       cy.location("pathname").should("eq", "/logout");
```
Comment (ti2ger92) [inline tests/e2e/specs/login-tests.js]: When I ran the test like this, it failed apparently because of a timeout issue, so I switched to a test of the page its self with a timeout. Does this pass consistently for you when you execute: 'nom run e2e', and click the test?

Comment (StewartWallace1115) [reply tests/e2e/specs/login-tests.js]: I believe I was able to recreate the issue on my machine. I fetch the logout link now instead of trying to search and find the element containing the text log out. I will push the updated code tomorrow. When I submit my new PR, do you mind testing it out and see if it works on your computer? I just want to make sure. Thank you.

Comment (jkeat) [reply tests/e2e/specs/login-tests.js]: @StewartWallace1115 @ti2ger92 any update here?

Comment (StewartWallace1115) [reply tests/e2e/specs/login-tests.js]: @jkeat  @ti2ger92 Just updated it. Can someone pull and try it out, please?

Comment (ti2ger92) [reply tests/e2e/specs/login-tests.js]: 2 of 3 passed at first. After several reruns, more logouts failed. This is why I changed the success criteria for a completed logout from the url to an element on the page on my on PR, because that can have a timeout configured.

Screen Shot 2020-01-08 at 8 28 07 PM

Comment (StewartWallace1115) [reply tests/e2e/specs/login-tests.js]: @ti2ger92 Can you try it again when you a chance, please? If it fails, can you post a screenshot as you did before? Thank you.

Comment (ti2ger92) [reply tests/e2e/specs/login-tests.js]: Hi! Works! Thanks Stewart for accommodating my Docker dreams.

I WAS able to get it to fail once like this:
1. fire everything up
2. run e2e test (pass)
3. immediately run test again (same failures as above).

However, once I gave the system a little time to settle, I got consistent positive tests. I say merge it!

Comment (jkeat) [reply tests/e2e/specs/login-tests.js]: I ran into the same issue and had to increase the timeout from 1 second to 3 seconds to make it pass -- I can merge this for now, but I feel like in general, it'd be best not to rely on timeouts too much
